src/components/OSCALSystemImplementation.test.js

```diff
@@ -2,72 +2,88 @@ import React from "react";
 import { render, screen, within } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import OSCALSystemImplementation from "./OSCALSystemImplementation";
-import testOSCALResponsibleRoles from "./OSCALResponsibleRoles.test";
 import { metadataTestData } from "../test-data/CommonData";
 import { systemImplementationTestData } from "../test-data/SystemData";
 
-function systemImplementationRenderer() {
-  render(
-    <OSCALSystemImplementation
-      systemImplementation={systemImplementationTestData}
-      parties={metadataTestData.parties}
-    />
-  );
-}
+describe("SystemImplementation", () => {
+  beforeEach(() => {
+    render(
+      <OSCALSystemImplementation
+        systemImplementation={systemImplementationTestData}
+        parties={metadataTestData.parties}
+      />
+    );
+  });
 
-export default function testOSCALSystemImplementation(
```

Comment:
In an effort to not continue adding to our less than optimal tests, I decided to go ahead and replace this with a `Describe`. It still tests the same thing, but in a more clear way.